ci-operator/step-registry/hypershift/mce/heterogeneous/power/create/nodepool/hypershift-mce-heterogeneous-power-create-nodepool-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
+  as: hypershift-mce-heterogeneous-power-create-nodepool
```

Comment:
this chain has only one step, so I think we currently don't need to define this chain.

Reply:
removed it, thanks!